packages/terra-dynamic-grid/src/styles.js

```diff
@@ -0,0 +1,110 @@
+// IE < 11 doesn't honor window.CSS.Supports. Edge has supports, but will fail on grid.
+const cssGridsEnabled = window.CSS && window.CSS.supports && window.CSS.supports('display', 'grid');
+
+const gridTemplateColumns = (layout) => {
+  const columns = layout['grid-template-columns'];
+  const msColumns = columns.split(/\s+/).join(` ${layout['grid-gap'] || 0} `);
```

Comment:
do we allow users to specify [grid-column-gap](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column-gap)?